src/Sulu/Bundle/AdminBundle/Resources/js/components/Grid/tests/Grid.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
```

Comment:
Oh, and by the way, we've decided to use flow also in tests, to avoid mocking classes in a wrong way. You can have a look at an example at https://github.com/sulu/sulu/pull/3607/files#diff-37fa534bf41281dea98a5fb1c89c0e49R1. Please update this test after the other PR is merged (there is some setup for this).

Reply:
I've just tried that on my own, and the flow checks are already failing here, because the `Grid.Item` has to have children. That already raises the question if a `Grid.Item` is allowed to be empty. If no we should also adjust the tests here, if yes we have to make the children attribute optional in the types.

I think I would say that we accept anything in a `Grid.Item` it should also be allowed to pass nothing, so I would go for adjusting the type.

And in order for that to work correctly you have to do a rebase.